### ISSUE TITLE
View contract transactions status

### ISSUE THREAD

**Issue by kborovik** [logictimes/PrivateKey.Space#13]
**As an (actor):**
user

**I want to (intention):**
be able to view transaction status

**So that (reason):**
- view transaction proposal quorum (accepted or rejected)
- view transaction broadcast and confirmation status
- view transaction notes

**Acceptance Criteria:**
- able to view transaction proposal quorum (accepted or rejected)
- able to view transaction broadcast and confirmation status
- able to view transaction notes

**Comment by alienatix** [comment]
> view in blockchain

do you mean it's an external link to https://blockchain.info ?

**Comment by kborovik** [comment]
@alienatix, yes services similar to blockchain.info, every web wallet offers an independent source to verify transaction status.
